src/ui/SurveyUnit/Identification/IdentificationCard.tsx

```diff
@@ -83,19 +87,20 @@ export function IdentificationCard({ surveyUnit }) {
   );
 }
 
-/**
- * @param {SurveyUnit} surveyUnit
- */
-function MoveQuestion({ surveyUnit }) {
+interface MoveQuestionProps {
+  surveyUnit: SurveyUnit;
+}
+
+function MoveQuestion({ surveyUnit }: Readonly<MoveQuestionProps>) {
   const options = [
     { label: D.yes, value: true },
     { label: D.no, value: false },
   ];
   const value = surveyUnit.move;
-  const handleChange = e => {
+  const handleChange = (e: { target: { checked: any; value: string } }) => {
```

Comment:
You can not use a out-of-the-box React type ? Like ChangeEventHandler . 

https://www.totaltypescript.com/event-types-in-react-and-typescript